src/Microsoft.ML.Maml/Microsoft.ML.Maml.csproj

```diff
@@ -3,12 +3,11 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DefineConstants>CORECLR</DefineConstants>
-    <IncludeInPackage>Microsoft.ML</IncludeInPackage>
```

Comment:
>ML [](start = 32, length = 2)

Microsoft.ML.Sweep? you need this dll for ResultProcessor.

Reply:
I would prefer to not put these in Sweep. Maybe another nuget package for execution? Microsoft.ML.Exec? This would include both maml and results processor.

Reply:
What if we just don't ship `Microsoft.ML.Sweeper` in any NuGet package for now, until someone makes a case that it must be shipped?

---
In reply to: [259190210](https://github.com/dotnet/machinelearning/pull/2690#discussion_r259190210) [](ancestors = 259190210)

Reply:
BTW - if there is a high level name for a package that can include sweep, results processor and maml. Im open to that as well.

Reply:
I double that, I don't see any reason to ship Sweeper right now.
I've went through code, and I don't see any entrypoints, so it's not consumable by Nimbus, and everything else we don't care right now.

---
In reply to: [259440526](https://github.com/dotnet/machinelearning/pull/2690#discussion_r259440526) [](ancestors = 259440526,259190210)